### PR TITLE
Feat upgrade docs

### PR DESCRIPTION
**Pull request by christyjacob4**

### CODE REVIEW

File: app/views/docs/installation.phtml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<div class="notice margin-bottom">
+    <h3>Migration Instructions</h3>
+    <p>If you are migrating from an older version of Appwrite, you need to follow the migration instructions <a href="/docs/upgrade" target="blank" rel="noopener">here.</a></p>
```
Comment (eldadfux) [inline app/views/docs/installation.phtml:3]:
```suggestion
    <p>If you are migrating from an older version of Appwrite, you need to follow the <a href="/docs/upgrade">migration instructions</a></p>
```

Better for SEO to have a meaningful text as the link. No need to for a new tab or noopener setting for a local link

File: app/views/docs/installation.phtml [hunk 2]
```diff
@@ -1,3 +1,8 @@
+<div class="notice margin-bottom">
+    <h3>Migration Instructions</h3>
```
Comment (eldadfux) [inline app/views/docs/installation.phtml:2]:
```suggestion
    <h3>Upgrading From Older Versions</h3>
```

File: app/views/docs/upgrade.phtml [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
 </div>
 
 <!-- <p>When updating to a new Appwrite version, make sure to run the <a href="/docs/installation">installation script for the new version</a> from the same location, you executed your original setup. This will allow the installation script to update your existing docker-compose.yml file. The correct location for execution is the parent directory of your existing appwrite directory.</p> -->
 
 <!-- <p>The data migration tool will allow you to easily migrate your current Appwrite data to work with the new version. </p> -->
 
-<p>To start the migration, run the following command from your terminal once your backup is ready and that your new Appwrite setup has been completed successfully. Replace "[APPWRITE_CONTAINER_ID]" with your container ID. You can find your container ID by running `docker ps` command.</p>
+<p> Head over to the directory containing your <b>docker-compose.yml</b> . If Appwrite is currently running, your first step would be to shut it down using: </p>
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
```
Comment (eldadfux) [inline app/views/docs/upgrade.phtml:14]: Why do we need to shut it down before setup? The install script already takes care of that

File: app/views/docs/upgrade.phtml [hunk 4]
```diff
@@ -1,19 +1,40 @@
...
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>docker ps | grep appwrite/appwrite</code></pre>
+</div>
+<p>Verify that the <b>STATUS</b> doesn't have any errors and all the <b>appwrite/appwrite</b> containers have version : <?php echo APP_VERSION_STABLE; ?></p>
+
+<p>We can now start the migration. Navigate to the <b>appwrite</b> directory where your <b>docker-compose.yml</b> is present and run the following command</p>
+
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>docker-compose exec appwrite migrate</code></pre>
+</div>
```
Comment (eldadfux) [inline app/views/docs/upgrade.phtml:40]: Maybe we should add an ending line? like once the setup has completed your Appwrite data is ready... you can now...

**Review by eldadfux — COMMENTED** [review]
I think we should also mention that the migration tool right now support migration between current version and previous version and you should avoid jumping multiple versions?

File: app/views/docs/upgrade.phtml (hunk 3 above)
Comment (christyjacob4) [reply app/views/docs/upgrade.phtml:14]: Okay I'll remove this

**Pull request merged**